pkg/loadbalancer/experimental/testdata/reuse.txtar

```diff
@@ -0,0 +1,92 @@
+#! --enable-experimental-lb --lb-test-fault-probability=0.0
+#
+# Test the reuse of a ClusterIP. Validates that we process the changes
+# in services in the right order even though the reflector is batching
+# events.
+#
+# This tests for the regression where the events were processed
+# in the wrong order, e.g. the creation of a new service before the deletion
+# of the old service, leading to "frontend already owned by another service"
+# error. Triggered easily with stress.sh:
+#
+#  === NAME  TestScript
+#    logger.go:256: ... msg="Failure processing services" error="frontend already owned by another service ..."
```

Comment:
just a question, would the reconciler eventually have taken care of this?

Reply:
no, this happens when the reflector is writing to the table, e.g. there's a conflict and the insert will fail.